cmd/gossamer/commands/account_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package commands
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestAccountGenerate test "gossamer account generate"
+func TestAccountGenerate(t *testing.T) {
```

Comment:
I can't help but notice all of these tests follow the same structure. I wonder if we would benefit from having one test that iterates through test cases to run. Something like [this](https://github.com/ChainSafe/gossamer/blob/91b748f54bf9a359e02be1edd5ab9add32170dce/dot/core/service_test.go#L125) is what I mean. Lets see if @timwu20 has any thoughts

Just a thought, definitely not something that is a must!

Reply:
ah okay I see. Theres already so much going on in this PR, I imagine this can be a later thing to update since its not super high priority

Reply:
These can be sub tests for the account command, but I can go eithe rway.